core/ustring.h

```diff
@@ -411,8 +411,10 @@ _FORCE_INLINE_ bool is_str_less(const L *l_ptr, const R *r_ptr) {
 // and doc translate for the class reference (DTR).
 #ifdef TOOLS_ENABLED
 // Gets parsed.
-String TTR(const String &);
-String DTR(const String &);
+String TTR(const String &p_text, const String &p_context = "");
+String TTRN(const String &p_text, const String &p_text_plural, int p_n, const String &p_context = "");
```

Comment:
You forgot to define the `TTRN` stub for the `#else` branch, so this will fail compiling `tools=no` once we start using the feature.